code/modules/reagents/reagent_containers/food/mobcube.dm

```diff
@@ -78,11 +78,15 @@
 		to_chat(M, "<span class='warning'>\The [src] expands!</span>")
 	if(!contained_mob)
 		return
+	var/mob/C = contained_mob
 	if(ispath(contained_mob))
-		new contained_mob(get_turf(src))
+		C = new contained_mob(get_turf(src))
 	else if(ismob(contained_mob))
-		var/mob/C = contained_mob
```

Comment:
... and I'd restore a ``C = contained_mob`` here. That way, we will only write a mob or null to the variable, and only as necessary.

Reply:
done